kubetest/e2e.go

```diff
@@ -493,6 +501,37 @@ func chartsTest() error {
 	return nil
 }
 
+func nodeTest(testArgs, nodeArgs []string, nodeTestArgs, project, zone string) error {
+	// Run node e2e tests.
+	// TODO(krzyzacy): remove once nodeTest is stable
+	if wd, err := os.Getwd(); err == nil {
+		log.Printf("cwd : %s", wd)
+	}
+
+	// prep node args
+	runner := []string{
+		"run",
+		fmt.Sprintf("%s/src/k8s.io/kubernetes/test/e2e_node/runner/remote/run_remote.go", os.Getenv("GOPATH")),
+		"--cleanup",
+		"--logtostderr",
+		"--vmodule=*=4",
+		"--ssh-env=gce",
+		"--results-dir=/src/k8s.io/kubernetes/_artifacts",
+		fmt.Sprintf("--project=%s", project),
+		fmt.Sprintf("--zone=%s", zone),
+		fmt.Sprintf("--ssh-user=%s", os.Getenv("USER")),
```

Comment:
We need to ensure the `USER` is set to non-root, otherwise, it won't work (at least) on Ubuntu. Maybe allow passing the name using some flag in `config.json`?